### PR TITLE
Update mailbox-usage.md

### PR DESCRIPTION
**Pull request by msbemba**
Corrected navigation to usage reports

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3368

File: microsoft-365/admin/activity-reports/mailbox-usage.md [hunk 1]
```diff
@@ -32,8 +32,7 @@ The **Mailbox usage report** provides information about users with a user mailbo
 
 1. In the admin center, go to the **Reports** \> <a href="https://go.microsoft.com/fwlink/p/?linkid=2074756" target="_blank">Usage</a> page.
 
-    
-2. From the **Select a report** drop-down, select **Exchange** \> **Mailbox usage**.
+2. Click on  **View More** under **Active users - Microsoft 365 Services** and Click on **Active users** drop down, select **Exchange** \> **Mailbox usage**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/mailbox-usage.md]:
```suggestion
2. Select **View More** under **Active users - Microsoft 365 Services**. 
3. From the **Active users** drop-down list, select **Exchange** \> **Mailbox usage**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - suggestions only.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks.

**Pull request merged**